### PR TITLE
Initial blog post + link to blog

### PR DESCRIPTION
**Pull request by Dinnerbone**
image

image

### CODE REVIEW

File: blog/_posts/2023-03-12-progress-report.markdown [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- Extreme Pamplona: [https://flasharch.com/en/archive/play/e36aac73914ec8672218317e000615d7](https://flasharch.com/en/archive/play/e36aac73914ec8672218317e000615d7)
+
+# Incredible progress in AVM2 (ActionScript 3) support!
+- Our website now has a page listing exactly what ActionScript 3 APIs we have implemented, making it easy to follow our progress. It will be frequently updated! [https://ruffle.rs/avm2.html](https://ruffle.rs/avm2.html)
+- XML support is rapidly improving! ActionScript 3 games tend to use a wide variety of XML methods. As Ruffle gains support for these methods, games are springing to life!
+- Several problems that caused unresponsive buttons and menus have been fixed. Unclickable buttons in ActionScript 3 games are (mostly) a thing of the past!
+- Other major refactors are in progress to improve Ruffle's compatibility with ActionScript 3 frameworks like Haxe.
+- @Aaron1011 is working on **Stage3D support**! Thanks to his work, the gorgeous interstellar strategy game Solarmax 2 will soon be playable in Ruffle! Check out the video below.
```
Comment (Aaron1011) [inline blog/_posts/2023-03-12-progress-report.markdown:26]: Since this was written, Solarmax2 became fully playable (at least on the desktop player).

Comment (n0samu) [reply blog/_posts/2023-03-12-progress-report.markdown:26]: Similarly, dynamic audio buffering is out now. So yeah, there's a few things that need to be rephrased

Comment (Dinnerbone) [reply blog/_posts/2023-03-12-progress-report.markdown]: How's that?

File: blog/_posts/2023-03-12-progress-report.markdown [hunk 2]
```diff
@@ -23,7 +23,7 @@ Thanks to a massive code refactor by @CUB3D, dozens upon dozens of ActionScript
 - XML support is rapidly improving! ActionScript 3 games tend to use a wide variety of XML methods. As Ruffle gains support for these methods, games are springing to life!
 - Several problems that caused unresponsive buttons and menus have been fixed. Unclickable buttons in ActionScript 3 games are (mostly) a thing of the past!
 - Other major refactors are in progress to improve Ruffle's compatibility with ActionScript 3 frameworks like Haxe.
-- @Aaron1011 is working on **Stage3D support**! Thanks to his work, the gorgeous interstellar strategy game Solarmax 2 will soon be playable in Ruffle! Check out the video below.
+- @Aaron1011 is working on **Stage3D support**! Thanks to his work, the gorgeous interstellar strategy game Solarmax 2 is now fully playable in Ruffle! Check out the video below.
```
Comment (Aaron1011) [inline blog/_posts/2023-03-12-progress-report.markdown:26]: Could you add 'in the desktop player' here? In my testing, Solarmax2 has some audio issues (and CORS issues, depending on the site) in the web version.

Comment (Dinnerbone) [reply blog/_posts/2023-03-12-progress-report.markdown]: Done!

**Pull request merged**